chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChipButtons.kt

```diff
@@ -0,0 +1,33 @@
+package com.decathlon.vitamin.compose.chips
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+
+object VitaminChipButtons {
+    @Composable
+    fun Close(
+        onClick: () -> Unit,
+        modifier: Modifier = Modifier,
+        contentDescription: String?,
```

Comment:
Should be before the `Modifier`.

Reply:
I removed onClick Callback because callback has to be on the whole Chip Component & not only the icon